logprep/processor/base/processor.py

```diff
@@ -122,7 +122,11 @@ class RuleBasedProcessor(BaseProcessor):
     def __init__(self, name: str, tree_config: str, logger: Logger):
         super().__init__(name, logger)
         self._rules = []
+        self._specific_rules = []
```

Comment:
where are these used?

Reply:
(commenting these out does not change anything about the test results)

Reply:
you have to define it in all childs. so it is better to define it here.

Reply:
ok.